docs/theming.md

```diff
@@ -0,0 +1,126 @@
+# Theming your Angular Material app
+
+
+### What is a theme?
+A **theme** is the set of colors that will be applied to the Angular Material components. The
+library's approach to theming is based on the guidance from the [Material Design spec][1]. 
+
+In Angular Material, a theme is created by composing multiple palettes. In particular, 
+a theme consists of:
+* A primary palette: colors most widely used across all screens and components.
+* An accent palette: colors used for the floating action button and interactive elements.
+* A "warn" palette: colors used to convey error state.
+* A foreground palette: colors for text and icons.
+* A background palette: colors used for element backgrounds.
+
+In Angular Material 2, all theme styles are generated _statically_ at build-time so that your
+app doesn't have to spend cycles generating theme styles on startup.
+
+[1]: https://material.google.com/style/color.html#color-color-palette
+
+### Using a pre-built theme
+Angular Material comes prepackaged with several pre-built theme css files. These theme files also
+include all of the styles for core (styles common to all components), so you only have to include a
+single css file for Angular Material in your app. 
+
+You can include a theme file directly into your application from 
+`@angular2-material/core/theming/prebuilt`
+
+If you're using Angular CLI, this is as simple as including one line
+in your `style.css`  file:
+```css
+@import '~@angular2-material/core/theming/prebuilt/deeppurple-amber.css';
+```
+
+Alternatively, you can just reference the file directly. This would look something like
+```html
+<link href="node_modules/@angular2-material/core/theming/prebuilt/indigo-pink.css" rel="stylesheet">
+``` 
+
+You can also concatenate the file with the rest of your application's css.
```

Comment:
I'll add that once we have a full complement of prebuilts (only three right  now)